<launch>
	 <!-- Logger config -->
  	<env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find driverless_test)/config/driverless_test.config"/>

	<node pkg="driverless_test" type="gps" name="gps_node" args="/dev/ttyUSB0" respawn="true"/>
	<node pkg="driverless_test" type="upper_control" name="upper_node" output="screen" respawn="true">

		<param name="speed" value="5"/>
	</node>
	<node pkg="driverless_test" type="lower_com" name="lower_node" args="/dev/ttyUSB1" output="screen" respawn="true"/>
</launch>
